qiita_db/environment_manager.py

```diff
@@ -191,30 +187,50 @@ def make_environment(load_ontologies, download_reference, add_demo_user):
 
     # Create the database
     print('Creating database')
+    create_settings_table = True
     admin_conn.autocommit = True
-    admin_conn.execute('CREATE DATABASE %s' % qiita_config.database)
+    try:
+        admin_conn.execute('CREATE DATABASE %s' % qiita_config.database)
+    except ValueError as error:
+        # if database exists, ignore if sett
+        if 'database "qiita_test" already exists' in error.message:
```

Comment:
This will only work if the database is named qiita_test ... perhaps it should look for:

```python
if 'database "%s" already exists' % qiita_config.database in error.message:
    ...
```